### PR TITLE
Add special installation instructions for stylized code component

### PR DESCRIPTION
**Pull request by gpleiss**
- Instructions for stylized code and terminal window only
- Includes installation instructions for `pui-prismjs` package
- Includes list of themes to install

### CODE REVIEW

File: src/pivotal-ui/components/code/code.scss [hunk 1]
```diff
@@ -43,6 +79,52 @@ name: code_styled
...
+
+    <li>
+      <p>Install a PrismJs theme. This makes the code look pretty.</p>
+      <div class="mbl">
+        <code class="pam">
+          <i class="fa fa-download" alt="Install Prismjs"></i>
+          npm install prismjs-&lt;theme&gt;-theme --save
+        </code>
+      </div>
+      <p>Some themes we recommend are:</p>
```
Comment (stubbornella) [inline src/pivotal-ui/components/code/code.scss]: The two light and dark Pivotal themes are:

File: src/pivotal-ui/components/code/code.scss [hunk 2]
```diff
@@ -130,52 +198,25 @@ end</code></pre>
...
-*/
-
-.pre-scrollable {
-  overflow-y: auto; //bootstrap override
-}
-
-/*doc
----
-title: Inline
-name: code_inline
```
Comment (stubbornella) [inline src/pivotal-ui/components/code/code.scss]: Why did these go away?

File: src/pivotal-ui/components/code/code.scss [hunk 3]
```diff
@@ -130,52 +198,25 @@ end</code></pre>
...
 
 /*doc
 ---
-title: Inline Dark
-name: code_inline_dark
-parent: code
----
-
-```html_example
-<p>This is an example of a paragraph with <code class="inline-dark">a dark inline code snippet</code> within it.</p>
```
Comment (stubbornella) [inline src/pivotal-ui/components/code/code.scss]: why did this go away?

File: src/pivotal-ui/components/code/code.scss (hunk 2 above)
Comment (matt-royal) [inline src/pivotal-ui/components/code/code.scss]: This text is still there... it was just moved higher.

File: src/pivotal-ui/components/code/code.scss (hunk 3 above)
Comment (matt-royal) [inline src/pivotal-ui/components/code/code.scss]: This text is still there... it was moved elsewhere in the file

**Pull request merged**